code/modules/surgery/organs/organ.dm

```diff
@@ -248,13 +248,18 @@ var/list/organ_cache = list()
 		return
 	switch (severity)
 		if (1.0)
-			take_damage(0,20)
+			take_damage(20,0)
```

Comment:
Organs don't currently support multiple types of damage. It's a type-less damage that is treated by the same surgery/medicine, rather than via specific surgery or medicines.

For the internal organ's take_damage proc, the arguments are damage and silent. Only external organs have arguments for different damage types.